labs.yml

```diff
@@ -128,6 +128,8 @@ commands:
     flags:
       - name: aws-profile
         description: AWS Profile to use for authentication
+      - name: kms-key
+        description: (Optional) KMS Key to be specified for the UC roles
       - name: single-role
```

Comment:
While you're at it. Add the role_name and policy_name parameters.

Reply:
added